x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/connector/syncjob/ConnectorSyncJobIndexServiceTests.java

```diff
@@ -228,6 +228,7 @@ public void testDeleteConnectorSyncJob_WithMissingSyncJobId_ExpectException() {
         expectThrows(ResourceNotFoundException.class, () -> awaitDeleteConnectorSyncJob(NON_EXISTING_SYNC_JOB_ID));
     }
 
+    // TODO Here! Doesn't work
```

Comment:
Is this comment still valid? We have `@AwaitsFix` if you are stuck with this test (tho it should just work since we also have e2e yaml tests covering this case)

Reply:
Ah this was left behind, it works now